src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -1050,7 +1062,12 @@
         }
       },
 
-      "required": ["fullyQualifiedLogicalName"]
+      "required": [ "fullyQualifiedLogicalName" ],
+      "dependencies": {
+        "uriBaseId": [ "uri" ],
```

Comment:
Not sure what you mean by "very root", but... keeping in mind that the value `uriBaseId` is an arbitrary string upon the meaning of which the producer and consumer of the log file agree... are you asking what would happen if the producer and consumer agreed that the meaning of, say `"target"` was the analysis target itself, so that the consumer would resolve `uriBaseId` to `file:///C:/bin/foo.dll"?